test/caching-precompiler.js

```diff
@@ -145,7 +146,7 @@ test('uses babelConfig for babel options when babelConfig is an object', functio
 	t.true('inputSourceMap' in options);
 	t.false(options.babelrc);
 	t.same(options.presets, ['stage-2', 'es2015']);
-	t.same(options.plugins, [customPlugin, powerAssert, rewrite, transformRuntime]);
+	t.same(options.plugins, [customPlugin, powerAssert, throwsHelper, rewrite, transformRuntime]);
```

Comment:
Oh - Understood. Still not a problem though. `power-assert` wraps inline, just like `ava-throws-helper`.

```js
t.true(_rec._expr(_rec._capt(_rec._capt(a, 'arguments/0/left') === 'bar', 'arguments/0'), {
	content: 't.true(a === \'bar\')',
	filepath: 'test/fixture/power-assert.js',
	line: 6
}));
```

Also, we don't enhance `t.throws` with `power-assert`.